packages/server/tests/acceptance/rpc_batch3.spec.ts

```diff
@@ -1012,6 +1012,8 @@ describe('@api-batch-3 RPC Server Acceptance Tests', function () {
       requestId = Utils.generateRequestId();
       reverterContract = await servicesNode.deployContract(reverterContractJson);
       // Wait for creation to propagate
+      await new Promise((r) => setTimeout(r, 7000));
```

Comment:
Not a fan of timeouts, also at this point no value in changing a release branch with this logic.